nncf/openvino/graph/model_transformer.py

```diff
@@ -354,9 +354,11 @@ def _set_const_value(node_with_const: ov.Node, const_port_id: int, const_value:
         if const_node is None:
             raise RuntimeError("Constant node was expected but could not find it.")
 
-        const_shape = const_node.get_data().shape
-        const_value = np.reshape(const_value, const_shape)
-        new_const_node = opset.constant(const_value, dtype=const_node.get_element_type())
+        const_shape = const_node.data.shape
+        const_dtype = const_node.data.dtype
+        const_value = np.reshape(const_value, const_shape).astype(const_dtype)
+
+        new_const_node = ov.op.Constant(const_value, shared_memory=True)
```

Comment:
What's wrong with the `opset.constant` method?

Reply:
Unfortunately, there is no `shared_memory` parameter in `opset.constant()`. As far as I understand, this parameter will be added in the future.

Reply:
Yes, but we use opset9 as opset (`from openvino.runtime import opset9 as opset`). Do you suggest using `opset13` only for constants or changing the opset in the import?

Reply:
`opset9` is quite outdated as we can see, so we can update it to `opset13` and fix the problems, IMO.
However, there is already an example of how to live with the different opsets - https://github.com/openvinotoolkit/nncf/blob/develop/tests/openvino/native/models.py#L19 (via #2268)

Reply:
Thanks! I've updated the code.